.travis.yml

```diff
@@ -63,8 +63,7 @@ before_script:
   # - ccache -s -z
   # Exit immediately if any unexpected error occurs.
   - set -e
-  - wget https://sourceforge.net/projects/freemarker/files/freemarker/2.3.8/freemarker-2.3.8.tar.gz/download -O freemarker.tgz
-  - tar -xzf freemarker.tgz freemarker-2.3.8/lib/freemarker.jar --strip=2
+  - wget https://ci.eclipse.org/openj9/userContent/freemarker.jar
```

Comment:
Any reason to remove the version information from the file name?

Reply:
Not really, and I was just thinking of putting that back in to make it obvious.  You're comment seals it.